packages/backend/src/modules/attachment/attachment.controller.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express'
+import { AttachmentDTO, CreateAttachmentDTO } from 'shared'
+import { Attachment } from './attachment'
+import { AttachmentService, attachmentService } from './attachment.service'
+import { AttachmentMapper, attachmentMapper } from './attachment.mapper'
+
+class AttachmentsController {
```

Comment:
add new method downloadFile where in request there is fileId, and in the response there is file